pkg/util/procfs/procfs.go

```diff
@@ -56,3 +61,46 @@ func (pfs *ProcFS) GetFullContainerName(pid int) (string, error) {
 	}
 	return containerNameFromProcCgroup(string(content))
 }
+
+func PidOf(name string) []int {
+	pids := []int{}
+	filepath.Walk("/proc", func(path string, info os.FileInfo, err error) error {
+		base := filepath.Base(path)
+		// Traverse only the directories we are interested in
+		if info.IsDir() && path != "/proc" {
+			// If the directory is not a number (i.e. not a PID), skip it
+			if _, err := strconv.Atoi(base); err != nil {
+				return filepath.SkipDir
+			}
+		}
+		if base != "cmdline" {
+			return nil
+		}
+		cmdline, err := ioutil.ReadFile(path)
+		if err != nil {
+			glog.V(10).Infof("Error reading file %s: %+v", path, err)
```

Comment:
nit: `V(4)` should be sufficient (https://github.com/kubernetes/kubernetes/blob/master/docs/devel/logging.md)